src/desktop/src/libs/realm.js

```diff
@@ -0,0 +1,27 @@
+/* global Electron */
+import base64 from 'base64-js';
+import { randomBytes } from './crypto';
+
+const ALIAS_REALM = 'realm_enc_key';
+/**
+ * Gets encryption key for realm.
+ * - Checks keychain if there is already an encryption key stored for realm data.
+ * - Returns encryption key stored in keychain (if exists).
+ * - Otherwise generates a new one and stores it in keychain.
+ *
+ * @method getEncryptionKey
+ * @returns {Promise}
+ */
+export const getEncryptionKey = () => {
```

Comment:
@rihardsgravis Please let me know if you have different ideas about organisation of this function. This could also be placed with the other utils but I kept it separate to follow the mobile pattern.